crates/net/network/src/peers.rs

```diff
@@ -325,33 +355,24 @@ impl PeersManager {
                 peer.state = PeerConnectionState::In;
 
                 is_trusted = is_trusted || peer.is_trusted();
-
-                // if a peer is not trusted and we don't have capacity for more inbound connections,
-                // disconnecting the peer
-                if !is_trusted && !has_in_capacity {
-                    self.queued_actions.push_back(PeerAction::Disconnect {
-                        peer_id,
-                        reason: Some(DisconnectReason::TooManyPeers),
-                    });
-                }
```

Comment:
what happened here?

Reply:
sorry, this logic is duplicated, I should modify them in another PR, not in this one.